### PR TITLE
CDS21 update for Privacy Sandbox

### PR DESCRIPTION
**Pull request by heyawhite**
Changes proposed in this pull request:

- Add the CDS21 update to the Privacy Sandbox docs

### CODE REVIEW

File: site/en/docs/privacy-sandbox/cds21-update/index.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
+---
+layout: layouts/doc-post.njk
+title: CDS 2021 Update
```
Comment (mihajlija) [inline site/en/docs/privacy-sandbox/cds21-update/index.md:3]:
```suggestion
title: Chrome Developer Summit 2021 update
```

File: site/en/docs/privacy-sandbox/cds21-update/index.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+
+{% Img src="image/VbsHyyQopiec0718rMq2kTE1hke2/Rg4lEQVzb1v6CZTyzFTE.png",
+alt="In 2019 we proposed PIGIN, followed by TURTLEDOVE in 2020 and FLEDGE in 2021.",
+width="800", height="169" %}
+
+There have also  been more than half a dozen other proposals offered by other
+companies, in the same solution space. Through continued collaboration, we hope
+to define a path forward. 
+
+At the same time, we’re starting [developer testing](https://developer.chrome.com/docs/privacy-sandbox/fledge/) for the
```
Comment (mihajlija) [inline site/en/docs/privacy-sandbox/cds21-update/index.md:131]:
```suggestion
At the same time, we’re starting [developer testing](/docs/privacy-sandbox/fledge/) for the
```

**Review by mihajlija — APPROVED** [review]

**Comment by rowan-m** [comment]
Looking good! Will take a final peek when the preview is up.

**Review by rowan-m — APPROVED** [review]
LGTM, visual changes look like a false positive.

**Pull request merged**